cloud-common/src/main/java/com/sequenceiq/cloudbreak/util/ClientUtil.java

```diff
@@ -75,6 +74,23 @@ public static Client createClient(String serverCert, String clientCert, String c
         return client;
     }
 
+    public static Client createClient() {
```

Comment:
I got that, but it takes a bunch of certificate related params, but I can merge them a handle nulls.